en/rocky/8/admin_guide/01-presentation.md

```diff
@@ -0,0 +1,299 @@
+
+UNIX is an open and evolving operating system that has played a major role in the history of computing. It has been the basis for many other systems: Linux, BSD, Mac OSX, etc.
+
+UNIX is still relevant today (HP-UX, AIX, Solaris, etc.)
+
+#### Minix
+
+* **1987**: A.S. Tanenbaum develops MINIX, a simplified UNIX, to teach operating systems in a simple way. Mr. Tanenbaum makes the source code of his operating system available.
+
+#### Linux
+
+* **1991**: A Finnish student, **Linus Torvalds**, creates an operating system dedicated to his personal computer and names it Linux. He publishes his first version 0.02, on the Usenet discussion forum and other developers come to help him improve his system. The term Linux is a play on words between the founder's first name, Linus, and UNIX.
+
+* **1993**: The Debian distribution is created. Debian is a non-commercial, community-based distribution. Originally developed for use on servers, it is particularly well suited for this role, but it is intended to be a universal system and thus usable on a personal computer as well. Debian is used as the basis for many other distributions, such as Mint or Ubuntu.
+
+* **1994**: The commercial distribution RedHat is created by the company RedHat, which is today the leading distributor of the GNU/Linux operating system. RedHat supports the community version Fedora and recently the free distribution CentOS.
+
+* **1997**: The KDE desktop environment is created. It is based on the Qt component library and the C++ development language.
+
+* **1999**: The Gnome desktop environment is created. It is based on the GTK+ component library.
+
+* **2002**: The Arch distribution is created. Its particularity is to be released in Rolling Release (continuous update).
+
+* **2004**: Ubuntu is created by the Canonical company (Mark Shuttleworth). It is based on Debian, but includes free and proprietary software.
+
+* **2021**: Birth of Rocky Linux, based on RedHat distribution.
+
+### Market share
+
+<!--
+TODO: graphics with marketshare for servers and pc.
+-->
+Linux is still not well known by the general public, even though they use it regularly. Indeed, Linux is hidden in **smartphones**, **televisions**, **internet boxes**, etc. Almost **70% of the web pages** served in the world are served by a Linux or UNIX server!
+
+Linux equips a little more than **3% of personal computers** but more than **82% of smartphones**. **Android** being an operating system whose kernel is a Linux.
+<!-- TODO: review those stats -->
+
+Linux equips 100% of the 500 supercomputers since 2018. A supercomputer is a computer designed to achieve the highest possible performance with the techniques known at the time of its design, especially with regard to computing speed.
+
+### Architectural design
+
+* The **kernel** is the first software component.
+ * It is the heart of the UNIX system.
+ * It manages the hardware resources of the system.
```

Comment:
I think you should include "Linux" here. If you want to include UNIX, that's OK too, but this is an educational document that focuses on Linux, and both UNIX and Linux share the "kernel" relationship. Consider: "It is the heart of a UNIX or Linux system."